examples/python/pytorch/resnet152_DDP_training.py

```diff
@@ -8,49 +8,52 @@
 import torch.multiprocessing as mp
 import torchvision.transforms as transforms
 import torchvision.models as models
+
 import time
 import pandas as pd
 
 
 def setup(rank, world_size):
-    os.environ['MASTER_ADDR'] = 'localhost'
-    os.environ['MASTER_PORT'] = '12355'
+    print("Running init_process_group...")
     dist.init_process_group("nccl", rank=rank, world_size=world_size)
+    print("Finished init_process_group...")
 
 def cleanup():
     dist.destroy_process_group()
 
-def train(local_rank, args):
-    rank = args.nr * args.gpus + local_rank	
+def train(gpu, args):
```

Comment:
the variable name "gpu" is vague. Perhaps rename to gpu_id?

Reply:
> the variable name "gpu" is vague. Perhaps rename to gpu_id?

https://medium.com/@auro_227/timing-your-pytorch-code-fragments-e1a556e81f2